lib/suc.libjsonnet

```diff
@@ -7,6 +7,9 @@ local params = inv.parameters.system_upgrade_controller;
   Plan(name, channel, label_selectors, concurrency, image, push_gateway, command): kube._Object('upgrade.cattle.io/v1', 'Plan', name) {
     metadata+: {
       namespace: params.namespace,
+      annotations+: {
+        'argocd.argoproj.io/sync-wave': '5',
```

Comment:
How was '5' chosen as the value? Just curious if we already have defined a set of sync-waves to use.

Reply:
Well did just choose something which still leave some place for other waves if required. Just like the good old BASIC method of keep some numbers free for further use.

To my knowledge, we haven't defined overall rules yet.

Reply:
All good for me, was just curious